salt/modules/mac_user.py

```diff
@@ -284,6 +292,11 @@ def chhome(name, home):
     # dscl buffers changes, sleep 1 second before checking if new value
     # matches desired value
     time.sleep(1)
+
+    # If persist, move the home directory contents to the new location
+    if persist:
+        __salt__['cmd.run'](['mv', pre_info['home'], home])
```

Comment:
If you `mv` a directory, it will move that directory _into_ the target directory. I'd expect this to move /home/OldShane to /home/NewShane/OldShane. Is that what you intended?